lib/buttons_bar.dart

```diff
@@ -0,0 +1,66 @@
+import 'plain_text.dart';
+import 'ui_constants.dart';
+import 'package:flutter/material.dart';
+
+
+class ButtonsBar extends StatefulWidget {
+  const ButtonsBar({super.key});
+
+  @override
+  State<ButtonsBar> createState() => _ButtonsBar();
+}
+
+class _ButtonsBar extends State<ButtonsBar> {
+
+  int _counter = 0;
+  final String playTrailer = 'Play trailer';
```

Comment:
Minor improvement, not PR blocker: You could use `static const` for this kind of constants!

Reply:
done